### ISSUE TITLE
Support for BC dates?

### ISSUE THREAD

**Issue by javiabellan** [closeio/ciso8601#156]
Are BC dates supported?

I try to 
```python
ciso8601.parse_datetime('-0399-01-01')
ciso8601.parse_datetime('0399-01-01 BC')
```

but those give me errors

**Comment by movermeyer** [comment]
Python's Date times do not support BCE dates by default.

https://docs.python.org/3/library/datetime.html

So probably not.

**Comment by movermeyer** [comment]
I agree, that we could give better error messages for the first case :thumbsup: good idea.

FWIW, the latter is not valid ISO 8601, so there won't be a better error message for that.
